ArcGISRuntimeSDKQt_CppSamples/Maps/setMaxExtent/SetMaxExtent.h

```diff
@@ -0,0 +1,67 @@
+// [WriteFile Name=SetMaxExtent, Category=Maps]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef SETMAXEXTENT_H
+#define SETMAXEXTENT_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class Map;
+class MapQuickView;
+class Envelope;
+class Graphic;
+class GraphicsOverlay;
+class SimpleLineSymbol;
```

Comment:
Our samples are inconsistent in this, but what do you think about alphabetizing this list?